website/docs/r/linux_virtual_machine.html.markdown

```diff
@@ -156,6 +156,8 @@ The following arguments are supported:
 
 * `plan` - (Optional) A `plan` block as defined below. Changing this forces a new resource to be created.
 
+* `platform_fault_domain` - (Optional) Specifies the virtual machine scale set logical fault domain into which this Linux Virtual Machine will be created. Defaults to `-1`. By specifying the default value, this Linux Virtual Machine will be automatically assigned to a fault domain that best maintains balance across available fault domains.
```

Comment:
we can simplify this to (and mark this as ForceNew):

```suggestion
* `platform_fault_domain` - (Optional) Specifies the Platform Fault Domain in which this Linux Virtual Machine should be created. Defaults to `-1`, which means this will be automatically assigned to a fault domain that best maintains balance across the available fault domains. Changing this forces a new Linux Virtual Machine to be created.
```